src/app/views/query-response/adaptive-cards/AdaptiveCard.tsx

```diff
@@ -91,6 +91,10 @@ const AdaptiveCard = (props: any) => {
     try {
       adaptiveCard.parse(data.card);
       const renderedCard = adaptiveCard.render();
+      if(renderedCard){
+        renderedCard.style.backgroundColor = currentTheme.palette.whiteTranslucent40;
+        renderedCard.style.color = 'black';
```

Comment:
Shouldn't this too get it's colour from the currentTheme palette?

Reply:
Yes. @ElinorW pointed this out and I already have a fix out